### PR TITLE
fix: keep same class fields sort

### PR DESCRIPTION
**Pull request by fengmk2**
pick from https://github.com/node-modules/hessian.js/pull/61

### CODE REVIEW

File: lib/v2/encoder.js [hunk 1]
```diff
@@ -40,6 +41,7 @@ var proto = Encoder.prototype;
 proto.reset = proto.clean = function () {
   EncoderV1.prototype.clean.call(this);
   this._classRefs = [];
+  this._classRefFields = {};
```
Comment (fengmk2) [inline lib/v2/encoder.js]: master 忘记 reset 了，我再提一个 pr

Comment (fengmk2) [reply lib/v2/encoder.js]: https://github.com/node-modules/hessian.js/pull/87

**Comment by dead-horse** [comment]
这边没问题合并之后我再修测试

**Comment by fengmk2** [comment]
@dead-horse 你先修复，我再 rebase

**Comment by fengmk2** [comment]
1.4.2

**Pull request merged**